themes/default/content/docs/reference/pulumi-yaml.md

```diff
@@ -8,49 +8,121 @@ menu:
-    - `refresh`: (optional) (`string`) set to `always` to refresh the state of the stack's resources before the deployment operations (e.g. `up`, `pre`, `destroy`, etc.). This is equivalent to passing the `--refresh=true` command-line flag. Passing `--refresh=false` will override the option set in `Pulumi.yaml`.
+The runtime attribute has an additional optional `options` option where you can specify additional runtime configuration.
 
-- `template`: (optional) provides configuration settings that will be used when initializing a new stack from a project file using `pulumi new`. Currently these values are *only- used by `pulumi new`, and not by `pulumi stack init` or as default configuration for existing stacks.
-    - `description`: (optional) a description for the template itself.
-    - `config`: (required) the map of configuration values keyed by the name of the config setting - such as `aws:region`.  The value of each key includes:
-        - `description`: (optional) a description for the config setting.
-        - `default`: (optional) the default value of the config setting - which will be presented to the user as a default.
-        - `secret`: (optional) if `true` indicates that this configration value should be marked as secret.
+| Name | Use case | Description |
+| - | - | - |
+| `typescript` | Only applicable for the nodejs runtime | Boolean indicating whether to use `ts-node` or not. |
+| `binary` | Applicable for the go, .net, and java runtimes | Path to pre-built executable. |
+| `virtualenv` | Ony applicable for the python runtime | Virtual environment path. |
+| `compiler` | Only applicable for YAML projects | Executable and arguments that emit to standard out. |
 
-- `plugins`: (optional) allows the overriding of plugin selection to make use of plugins not installed into the global plugin cache.
-  - `providers`/`analyzers`/`languages`: (optional) each entry has a list of plugins underneath it.
-    - `name`: (required) the name of the plugin.
-    - `path`: (required) the path to the folder containing the plugin.
-    - `version`: (optional) the version of the plugin, if not set this plugin will match for any version the engine requests.
+#### About `binary`
+
+- For Go, if not specified, go sources in $CWD will be invoked via `go run`.
+- For .NET, f not specified, a .NET project in $CWD will be invoked via `dotnet run`.
+
+#### About `virtualenv`
+
+New Python projects created with `pulumi new` have this option set by default. If not specified, Pulumi will invoke the `python3` command it finds on $PATH (falling back to `python`) to run the Python program. To use a virtual environment without the `virtualenv` option, run `pulumi` commands (such as `pulumi up`) from an activated virtual environment shell. Or, if using a tool like [Pipenv](https://github.com/pypa/pipenv), prefix `pulumi` commands with `pipenv run pulumi ...`.
+
+### `options` options
+
+| Name | Required | Description | Default |
```

Comment:
Debatable whether a table is more readable when there's a single line. (Not sure myself.) But I get its inclusion for consistency.

Reply:
gonna leave as is, for now